### ISSUE TITLE
pdftotext -layout

### ISSUE THREAD

**Issue by MikhailKlemin** [sajari/docconv#39]
Hello! 
I woud use `-layout` option of pdftotext for that  I guess I have to change 
`body, err := exec.Command("pdftotext", "-q", "-nopgbrk", "-enc", "UTF-8", "-eol", "unix", f.Name(), "-").Output()` to add `-layout` am I correct?

**Comment by mish15** [comment]
@MikhailKlemin yes you would, however it may be worth making that default for all. Do you have any text examples showing the difference with and without the `layout` option?

**Comment by MikhailKlemin** [comment]
Hi
For me it makes a lot of sense, since usually I apply a lot of regex after converting to TXT, and `-layout ` really help to fight the mess. I attached an example with screenshots. 
![with_layout](https://user-images.githubusercontent.com/8426672/32363732-c0f0d714-c081-11e7-8383-4beee4ad6ffb.png)
![no_layout](https://user-images.githubusercontent.com/8426672/32363733-c10cf016-c081-11e7-9337-23977a26b5da.png)
![pdf](https://user-images.githubusercontent.com/8426672/32363734-c1265a42-c081-11e7-946a-b538490cf8fc.png)

**Comment by mish15** [comment]
@MikhailKlemin we normally have to clean up the whitespace, so we'd need to test this internally to see what happens. I think it's worth adding as an option. I would look at adding some ENV options to control this. What's your timeframe?
